app/Filament/Server/Pages/Console.php

```diff
@@ -47,6 +53,38 @@ public function mount(): void
         }
     }
 
+    public function boot(): void
+    {
+        /** @var Server $server */
+        $server = Filament::getTenant();
+        /** @var FeatureProvider $feature */
+        foreach ($server->egg->features() as $feature) {
+            $this->cacheAction($feature->getAction());
+        }
+    }
+
+    #[On('mount-feature')]
+    public function mountFeature(string $data): void
+    {
+        $data = json_decode($data);
+        $feature = data_get($data, 'key');
+        $line = data_get($data, 'line');
+
+        Activity::event('server:feature')
```

Comment:
Why is there an activity log for this?!

Reply:
I think we can remove it.

Paging @rmartinoscar